Extension/src/LanguageServer/client.ts

```diff
@@ -897,7 +897,7 @@ export class DefaultClient implements Client {
                         this.disposables.push(vscode.languages.registerDocumentSymbolProvider(this.documentSelector, new DocumentSymbolProvider(this), undefined));
                         this.disposables.push(vscode.languages.registerCodeActionsProvider(this.documentSelector, new CodeActionProvider(this), undefined));
                         const settings: CppSettings = new CppSettings();
-                        if (settings.formattingEngine !== "Disabled") {
+                        if (settings.formattingEngine?.toLowerCase.toString() !== "disabled") {
```

Comment:
Similar to previous comment.  I'm not sure what `toLowerCase.toString()` results in, but I assume it's not returning what you expect.